### PR TITLE
OSDOCS-11712: Documented the 4.15.28 z-stream release notes

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.15

Issue:
[OSDOCS-11712](https://issues.redhat.com/browse/OSDOCS-11712)

Link to docs preview:
* [4.15.28](https://80632--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes.html#ocp-4-15-28_release-notes)

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -2733,6 +2733,33 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+[source,terminal]
+----
+$ oc adm release info 4.15.28 --pullspecs
+----
+
+[id="ocp-4-15-28-known-issues_{context}"]
+==== Known issues
+
+* An error might occur when deleting a pod that uses an SR-IOV network device. This error is caused by a change in {op-system-base} 9 where the previous name of a network interface is added to its alternative names list when it is renamed. As a consequence, when a pod attached to an SR-IOV virtual function (VF) is deleted, the VF returns to the pool with a new unexpected name, for example, `dev69`, instead of its original name, for example, `ensf0v2`. Although this error is non-fatal, Multus and SR-IOV logs might show the error while the system recovers on its own. Deleting the pod might take a few extra seconds due to this error. (link:https://issues.redhat.com/browse/OCPBUGS-38163[*OCPBUGS-38163*],link:https://issues.redhat.com/browse/OCPBUGS-18822[*OCPBUGS-18822*], link:https://issues.redhat.com/browse/RHEL-5988[*RHEL-5988*])
```
[the remark this answers is not in the document]
Comment (JoeAldinger) [reply release_notes/ocp-4-15-release-notes.adoc:2756]: I think this is what I suggested for 4.16.z this week. 😄

**Review by JoeAldinger — COMMENTED** [review]
Same suggestion as 4.16.z. Let me know when these ship and I'll merge.

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-15-release-notes.adoc]: These clones. Updated!

**Pull request merged**